### PR TITLE
Dynamic team page

### LINKED ISSUE #76: Inject models into team page
Now that we have the frontend and backend of this page ready we can put them together!

### PR DESCRIPTION
**Pull request by toni-santos**
Closes #76

### CODE REVIEW

**Comment by toni-santos** [comment]
image

I limited the social media to those that seem the most relevant, furthermore, they only appear on hover of the member.

**Comment by Naapperas** [comment]
> image
> 
> I limited the social media to those that seem the most relevant, furthermore, they only appear on hover of the member.

Nice catch. Furthermore, there should be some kind of contrast for the social links, regardless of the color of the background image.

**Comment by pedrosilva17** [comment]
> > image
> > I limited the social media to those that seem the most relevant, furthermore, they only appear on hover of the member.
> 
> Nice catch. Furthermore, there should be some kind of contrast for the social links, regardless of the color of the background image.

Agreed, maybe we could color the icons' background according to the team member's department, instead of the icons themselves. Or just slap a 2023-bg behind them

File: .env [hunk 1]
```diff
@@ -2,7 +2,7 @@ APP_NAME="SINF Website 2023"
 APP_ENV=local # Use "maintenance" to enable the maitenance page
 APP_KEY=base64:AVegBkyjHs2QQsx53M9Kt9GbnrDLc6yG4cRg+WMJrbA=
 APP_DEBUG=true
-APP_URL=http://localhost
+APP_URL=http://localhost:8000/
```
Comment (ttoino) [inline .env]: The slash at the end of this can cause problems

```suggestion
APP_URL=http://localhost:8000
```

**Review by ttoino — DISMISSED** [review]

**Comment by Naapperas** [comment]
Beware of the merge conflicts

**Review by pedrosilva17 — APPROVED** [review]

**Review by ttoino — APPROVED** [review]

**Pull request merged**